src/Codata/Musical/Colist/Bisimilarity.agda

```diff
@@ -8,8 +8,8 @@
 
 module Codata.Musical.Colist.Bisimilarity where
 
-open import Codata.Musical.Colist.Base
-open import Codata.Musical.Notation
+open import Codata.Musical.Colist.Base using (Colist; _∷_; []; map)
```

Comment:
```suggestion
open import Codata.Musical.Colist.Base using (Colist; []; _∷_; map)
```
It's always weird to see constructors mentioned in 'non-conventional' order, to me, at least.